modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/openapi/OpenAPIContext.java

```diff
@@ -19,6 +19,10 @@
  */
 public class OpenAPIContext {
 
+	public String getBaseUriInfo() {
```

Comment:
UriInfo is an object. Call this something else, like `baseUrl`. Otherwise it is confusing. I was expecting an object of type UriInfo.

Reply:
Make sense @4lejandrito!! I will reword it